atomics/t1170/t1170.yaml

```diff
@@ -0,0 +1,18 @@
+attack_technique: T1170
+display_name: Mshta
+
+atomic_tests:
+- name: Mshta executes JavaScript Scheme Fetch Remote Payload With GetObject
+  description: |
+    Test execution of a remote script using mshta.exe
+  supported_platforms:
+    - windows
+  input_arguments:
+    file_url:
+      description: location of the payload
+      type: Url
+      default: https://www.example.com/mshta.sct
```

Comment:
is this an SCT that should be committed into this repo in our atomic technique dir?

Reply:
I am not sure where our specs file went, but I thought the aim was to have the test scripts and payload in the same folder